src/server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const path = require('path');
+const { NOT_FOUND, INTERNAL_SERVER_ERROR } = require('http-status');
+
+const { apiV1Router } = require('./routes/api/v1');
+const { compressionMiddleware } = require('./middlewares');
+
+const app = express();
+const port = 3000;
+const frontendPath = path.join(
+  __dirname,
```

Comment:
If you keep your backend separated from frontend in 2 repos, consider splitting Express app into Angular serving and API ones, in a matter of 2 microservices working into single network